index.html

```diff
@@ -459,146 +459,53 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
   </section>
 
   <!-- Profiling mechanism -->
-
   <section id="profiling-mechanism">
-    <h2 id="the-profiling-profile">Profiling Mechanism</h2>
+    <h2>Profiling Mechanism</h2>
     <p>
-      This section describes a generic mechanism to define a
-      profile of the <a>WoT Thing Description</a> in a unambiguous way.
+      In order to denote that a given
+      <a href="https://www.w3.org/TR/wot-architecture/#dfn-thing">Web Thing</a>
+      conforms to a particular profile, its Thing Description
+      [[wot-thing-description]] MUST include a <code>profile</code> member whose
+      value is set to a valid profile identifier corresponding to that profile.
     </p>
     <p>
-      The W3C <a>WoT Thing Description</a> specification defines a
-      formal language, i.e. a set of vocabulary terms (keywords),
-      a set of classes that are built from these keywords, and a
-      set of additional rules, that define constraints on
-      permitted values and keyword presence (mandatory / optional)
-      dependent on the context where the keyword is used. In
-      addition the <a>WoT Thing Description</a> specification defines relationships and
-      corresponding cardinalities between these classes.
+      A profile identifier MUST be a valid URI [[RFC3986]].
     </p>
+    <pre class="example">
+    {
+      "@context": "https://www.w3.org/2019/wot/td/v1",
+      "id": "urn:dev:ops:32473-WoTLamp-1234",
+      "profile": "https://www.w3.org/2021/wot/profile/core",
```

Comment:
profile term does not exist yet, but I would be ok with `profile`. 
https://github.com/w3c/wot-thing-description/issues/806

Reply:
still we need a good name for "core"

Reply:
I suggest "Common Profile" https://github.com/w3c/wot-profile/issues/21#issuecomment-806543255

Reply:
@sebastiankb: If `profile` is defined in the Thing Description specification, do you think this section is still needed in the Profile specification? Or should the Profile specification just refer to the relevant section of the Thing Description specification in its introduction for example?

Is anyone working on a PR for https://github.com/w3c/wot-thing-description/issues/806 yet?

Reply:
yes, I will do it until the next TD meeting. That means we will have it available for the next WD release.

Reply:
@sebastiankb 
Please add a link to the PR that introduces the "profile" identifier in the TD spec to give context.

Reply:
I've added a link to the definition of the profile member in the TD specification now that has landed.